docs/key-formats.md

```diff
@@ -118,20 +119,48 @@ This currently happens in the core, but should move to the Bitcoin plugin.
 
 ### Public keys
 
+The public key format includes one or more public keys encoded according to BIP 32 "xpub" format:
+
 ```typescript
 interface PublicBitcoinKey {
-  bitcoinXpub: string,
-  format?: 'bip32' | 'bip44' | 'bip49' | 'bip84',
-  coinType?: number
+  publicKeys: {
+    bip32?: string
+    bip44?: string
+    bip49?: string
+    bip84?: string
+  }
 }
 ```
 
-The derivation path for `bitcoinXpub` depends on the format:
+Each of these keys are optional, and wallets should search for funds on whichever keys are present (in any combination):
+
+- bip32
+  - Receiving & change branch: xpub/0/n
+  - Script: p2pkh
+- bip44
+  - Receiving branch: xpub/0/n
+  - Change branch: xpub/1/n
+  - Script: p2pkh
+- bip49
+  - Receiving branch: xpub/0/n
+  - Change branch: xpub/1/n
+  - Script: segwit p2wpkh embedded in p2sh
+- bip84:
+  - Receiving branch: xpub/0/n
+  - Change branch: xpub/1/n
+  - Script: segwit p2wpkh
+
+It is customary for different branches to use different encoding prefixes (xpub vs ypub vs zpub), but this is not mandatory. Wallets should interpret each public key according to the table above, regardless of the encoding metadata used.
```

Comment:
As a "noob", I think being clear about the use of the word "xpub" is useful. Because xpub is interchangeably used as ether the "extended public key" no matter the serialization format and the serialization format, perhaps we should clarify that what we mean here is an extended public key either by use of another name in the derivation paths (perhaps `eXpub/0/n`, `xpubkey/0/n`, `pubkey/0/n`, `key/0/n`, or `k/0/n`). We can then mention that what this identifier means: an extended public key of any of the serialization formats (xpub, ypub, zpub, etc).

Reply:
> each public key according to the table above,

nit:

> each extended public key according to the table above,